content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Istio Webhook Management
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 10
+keywords: [security,webhook]
+---
+
+{{< warning >}}
+The following information describes an experimental feature, which is intended
+for evaluation purposes only.
+{{< /warning >}}
+
+This task shows how to how to manage webhooks in Istio through `istioctl`.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
```

Comment:
nit: I don't think this is needed

Reply:
Creating a new cluster is not strictly needed, but to on the safe side -> avoid affecting an existing cluster.